### PR TITLE
BR-155 Add Jenkinsfile to DDF-UI

### PR DESCRIPTION
**Pull request by TonyMorrison**
*** What does this PR do?
Add Jenkinsfile to ddf-ui

*** Reviewers
@andrewkfiedler 
@shaundmorris 

*** Testing
http://jenkins.phx.connexta.com/service/jenkins/job/ddf-ui/job/master/

### CODE REVIEW

**Review by andrewkfiedler — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,112 @@
+//"Jenkins Pipeline is a suite of plugins which supports implementing and integrating continuous delivery pipelines into Jenkins. Pipeline provides an extensible set of tools for modeling delivery pipelines "as code" via the Pipeline DSL."
+//More information can be found on the Jenkins Documentation page https://jenkins.io/doc/
+
+@Library('github.com/connexta/cx-pipeline-library@master') _
+@Library('github.com/connexta/github-utils-shared-library@master') __
+
+pipeline {
+    agent {
+        node {
+            label 'linux-large-ddf'
```
Comment (shaundmorris) [inline Jenkinsfile]: I would keep this large (based on what I know), but curious if has to be large

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        failure {
+            slackSend color: '#ea0017', message: "FAILURE: ${JOB_NAME} ${BUILD_NUMBER}. See the results here: ${BUILD_URL}"
+        }
+        unstable {
+            slackSend color: '#ffb600', message: "UNSTABLE: ${JOB_NAME} ${BUILD_NUMBER}. See the results here: ${BUILD_URL}"
+        }
+        cleanup {
+            catchError(buildResult: null, stageResult: 'FAILURE') {
+                echo '...Cleaning up workspace'
+                cleanWs()
```
Comment (shaundmorris) [inline Jenkinsfile]: ❔ Do we still need to explicitly clean up the workspace on failures?

**Review by shaundmorris — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (TonyMorrison) [reply Jenkinsfile]: it worked about as fast with linux-medium and linux-small so I changed to small

File: Jenkinsfile (hunk 2 above)
Comment (TonyMorrison) [reply Jenkinsfile]: removed

**Pull request merged**